rules/integrations/o365/initial_access_microsoft_365_impossible_travel_activity.toml

```diff
@@ -0,0 +1,49 @@
+[metadata]
+creation_date = "2021/07/15"
+maturity = "development"
+updated_date = "2021/07/15"
+
```

Comment:
Missing integration

Reply:
@w0rk3r theses rules have been on hold.

> It looks like I created these rules before elastic is able to trigger the detection. Should I just leave this open until the integration happens?
> 
> [elastic/integrations#1131](https://github.com/elastic/integrations/issues/1131) - @austinsonger 



> Up to you, we can change the maturity to `development` so they are in the repo and ready. Then later we can always"flip the switch" - @rw-access

Reply:
Yeah, we can merge them as `development` and switch as soon as the integration supports it